x/reward/abci.go

```diff
@@ -76,10 +69,16 @@ func handleKeyMgmtInflation(ctx sdk.Context, k types.Rewarder, m types.Minter, s
 	mintParams := m.GetParams(ctx)
 	totalAmount := minter.BlockProvision(mintParams).Amount.ToDec().Mul(k.GetParams(ctx).KeyMgmtRelativeInflationRate)
 
+	isProxyActive := func(v snapshot.ValidatorI) bool {
```

Comment:
At this point, the proper name for this function is more like `isParticipatingKeygen` instead of `isProxyActive`.
```suggestion
	isParticipatingKeygen := func(v snapshot.ValidatorI) bool {
```